frontend/src/app/modules/fields/edit/editing-portal/edit-form-portal.component.ts

```diff
@@ -17,9 +17,9 @@ import {
 } from "core-app/modules/fields/edit/edit-field.component";
 import {createLocalInjector} from "core-app/modules/fields/edit/editing-portal/edit-form-portal.injector";
 import {IFieldSchema} from "core-app/modules/fields/field.base";
-import {WorkPackageChangeset} from "core-components/wp-edit-form/work-package-changeset";
 import {EditFieldService, IEditFieldType} from "core-app/modules/fields/edit/edit-field.service";
-import {EditChangeset} from "core-app/modules/fields/changeset/edit-changeset";
+import {WorkPackageChangeset} from "core-components/wp-edit/work-package-changeset";
+import {ResourceChangeset} from "core-app/modules/fields/changeset/resource-changeset";
 import {HalResource} from "core-app/modules/hal/resources/hal-resource";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/b6e4680ef80bc97abdec363477c560842ceb1115/frontend/src/app/modules/fields/edit/editing-portal/edit-form-portal.component.ts#L24"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>